src/sql/workbench/parts/notebook/models/notebookModel.ts

```diff
@@ -777,7 +777,7 @@ export class NotebookModel extends Disposable implements INotebookModel {
 			}
 			await this.shutdownActiveSession();
```

Comment:
NotebookManager dispose() shutdowns all session again. they are dup. Should remove the dup?

Reply:
I found a couple of cases where dispose() wasn't actually being called and tested the scenario where we shutdown the session multiple times; the second one was basically a no-op.